### PR TITLE
Em 23

### PR DESCRIPTION
**Pull request by LuzLimon**

### CODE REVIEW

File: src/components/Dashboard/Dashboard.module.css [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+  .btnEmergency {
+    height: 80px;
+    width: 80px;
+    font-size: 25px;
+  }
+
+  .logo{
+    width: 200px;
+  }
+
```
Comment (FraGa96) [inline src/components/Dashboard/Dashboard.module.css]: remove extra space

File: src/components/Dashboard/Navbars/PsychologistNavbar.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	const [state, setState] = useState(false);
+
+	const asideOpenAction = () => {
+		if (!state) {
+			setState(true);
+		} else {
+			setState(false);
+		}
+	};
+	const Pshycologist = (
```
Comment (FraGa96) [inline src/components/Dashboard/Navbars/PsychologistNavbar.jsx]: is this a constant?
don't use PascalCase on constants
check the naming convention [https://github.com/lalodiazdiaz/Emotions/wiki/Code-Review](url)

**Review by FraGa96 — CHANGES_REQUESTED** [review]

**Review by FraGa96 — CHANGES_REQUESTED** [review]
Why every button is a submit button?

File: src/components/Dashboard/Dashboard.jsx [hunk 3]
```diff
@@ -1,20 +1,61 @@
...
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import styles from './Dashboard.module.css';
+import logo from '../../assets/logo.png';
+import btnMenu from '../../assets/menuIcon.png';
+import PatientLink from '../PatientLink/PatientLink';
+import PhichologistLink from '../PsychologistLink/PhichologistLink';
 
 function Dashboard() {
 	const loggedUser = window.localStorage.getItem('user');
```
Comment (lalodiazdiaz) [inline src/components/Dashboard/Dashboard.jsx:10]: check the naming convention [https://github.com/lalodiazdiaz/Emotions/wiki/Code-Review](https://github.com/lalodiazdiaz/Emotions/pull/url)

File: src/components/Dashboard/Dashboard.jsx [hunk 4]
```diff
@@ -1,20 +1,61 @@
...
 	const userLogged = JSON.parse(loggedUser);
+	const [state, setState] = useState(false);
+
+	const asideOpenAction = () => {
```
Comment (lalodiazdiaz) [inline src/components/Dashboard/Dashboard.jsx:14]: check the naming convention [https://github.com/lalodiazdiaz/Emotions/wiki/Code-Review](https://github.com/lalodiazdiaz/Emotions/pull/url)

File: src/components/Dashboard/Dashboard.module.css [hunk 5]
```diff
@@ -65,21 +54,6 @@ a {
...
-  font-size: 20px;
-  cursor: pointer;
-  border: 2px #000000 solid;
-}
-
-.btnEmergency:hover {
-  background-color: #ce3b4a;
-}
-
 @media (max-width: 1300px) {
```
Comment (lalodiazdiaz) [inline src/components/Dashboard/Dashboard.module.css:57]: image

identation  in asideMenuOption

File: .eslintrc.json [hunk 6]
```diff
@@ -35,6 +35,7 @@
         "null": "always"
       }
     ],
+    "react/jsx-closing-bracket-location": "off",
```
Comment (FraGa96) [inline .eslintrc.json]: don't disable eslint rules just like that, adapt the code, not the eslint rules

File: .eslintrc.json [hunk 7]
```diff
@@ -71,10 +72,7 @@
       "error",
       "tab"
     ],
-    "indent": [
-      "error",
-      "tab"
-    ],
+    "indent": 0,
```
Comment (FraGa96) [inline .eslintrc.json]: don't disable eslint rules just like that x2

File: src/components/Dashboard/Dashboard.module.css [hunk 8]
```diff
@@ -0,0 +1,174 @@
...
+
+  .resMenu {
+    display: flex;
+    position: fixed;
+    width: 100%;
+    height: 10%;
+    background-color: #8fe0d1;
+  }
+
+  .btnImage{
```
Comment (FraGa96) [inline src/components/Dashboard/Dashboard.module.css]:
```suggestion
  .btnImage {
```

File: src/components/Dashboard/Dashboard.module.css [hunk 9]
```diff
@@ -0,0 +1,174 @@
...
+    width: 15%;
+    background-color: #8fe0d1;
+    border: 0;
+  }
+
+  .contImage{
```
Comment (FraGa96) [inline src/components/Dashboard/Dashboard.module.css]:
```suggestion
  .contImage {
```

File: src/components/Dashboard/Dashboard.module.css [hunk 10]
```diff
@@ -0,0 +1,174 @@
...
+
+  a {
+    color: white;
+    text-align: center;
+    width: 100%;
+    font-size: 25px;
+    font-weight: bold;
+  }
+
+  .logo{
```
Comment (FraGa96) [inline src/components/Dashboard/Dashboard.module.css]:
```suggestion
  .logo {
```

File: src/components/PatientLink/PatientLink.jsx [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+			</Link>
+			<Link className={styles.linkNav}>
+				Evidencias
+			</Link>
+			<Link className={styles.linkNav}>
+				Perfil
+			</Link>
+		  </div>
+			<button className={styles.btnEmergency} type="button">911</button>
+		</>
```
Comment (FraGa96) [inline src/components/PatientLink/PatientLink.jsx]: align the code properly

**Review by FraGa96 — CHANGES_REQUESTED** [review]

**Review by lalodiazdiaz — DISMISSED** [review]

File: src/components/Dashboard/Dashboard.jsx [hunk 12]
```diff
@@ -1,9 +1,60 @@
...
+import logo from '../../assets/logo.png';
+import btnMenu from '../../assets/menuIcon.png';
+import PatientLink from '../PatientLink/PatientLink';
+import PhichologistLink from '../PsychologistLink/PhichologistLink';
 
 function Dashboard() {
+	const USER_LOGGED = JSON.parse(localStorage.getItem('user'));
+	const [state, setState] = useState(false);
+
+	const ASIDE_OPEN_ACTION = () => {
```
Comment (FraGa96) [inline src/components/Dashboard/Dashboard.jsx:13]: this is a function, why naming it like a constant?

File: src/components/PsychologistLink/PhichologistLink.module.css [hunk 13]
```diff
@@ -0,0 +1,39 @@
+.optionsAside {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  height: 200px;
+}
+
+.linkNav{
```
Comment (FraGa96) [inline src/components/PsychologistLink/PhichologistLink.module.css]:
```suggestion
.linkNav {
```

**Review by FraGa96 — CHANGES_REQUESTED** [review]

File: src/components/Dashboard/Dashboard.jsx (hunk 4 above)
Comment (FraGa96) [reply src/components/Dashboard/Dashboard.jsx:14]: this naming was correct in camelCase

File: src/components/Dashboard/Dashboard.jsx (hunk 3 above)
Comment (FraGa96) [reply src/components/Dashboard/Dashboard.jsx:10]: this naming was correct in camelCase

**Review by FraGa96 — APPROVED** [review]

**Review by MiguelAngelSF — APPROVED** [review]

**Pull request merged**